src/table/virtualized-table/__tests__/Table.spec.tsx

```diff
@@ -100,6 +103,10 @@ describe('<Table />', () => {
     isModal = false;
 
     constraints = {};
+
+    rootElement = {
+      getBoundingClientRect: () => ({ height: rect.width }),
```

Comment:
A bit confusing have the height equal the width. Why not do set it from `rect.height`?

Reply:
suppose to be height, fixed